### PR TITLE
shared key metadata support in notify

### PR DESCRIPTION
**Pull request by murali-shris**
**- What I did**
- Support for encryption shared key and public key in notify verb
**- How I did it**
- modified notify verb builder and notify verb syntax
**- How to verify it**
- run the unit test notify_verb_builder_test.dart

### CODE REVIEW

**Review by sitaram-kalluri — APPROVED** [review]

File: at_commons/lib/src/verb/syntax.dart [hunk 1]
```diff
@@ -29,7 +29,7 @@ class VerbSyntax {
...
       r'^stream:((?<operation>init|send|receive|done|resume))?((@(?<receiver>[^@:\s]+)))?( ?namespace:(?<namespace>[\w-]+))?( ?startByte:(?<startByte>\d+))?( (?<streamId>[\w-]*))?( (?<fileName>.* ))?((?<length>\d*))?$';
   static const notify =
-      r'^notify:((?<operation>update|delete):)?(messageType:(?<messageType>key|text):)?(priority:(?<priority>low|medium|high):)?(strategy:(?<strategy>all|latest):)?(latestN:(?<latestN>\d+):)?(notifier:(?<notifier>[^\s:]+):)?(ttln:(?<ttln>\d+):)?(ttl:(?<ttl>\d+):)?(ttb:(?<ttb>\d+):)?(ttr:(?<ttr>(-)?\d+):)?(ccd:(?<ccd>true|false):)?(@(?<forAtSign>[^@:\s]*)):(?<atKey>[^:@]((?!:{2})[^@])+)(@(?<atSign>[^@:\s]+))?(:(?<value>.+))?$';
+      r'^notify:((?<operation>update|delete):)?(messageType:(?<messageType>key|text):)?(priority:(?<priority>low|medium|high):)?(strategy:(?<strategy>all|latest):)?(latestN:(?<latestN>\d+):)?(notifier:(?<notifier>[^\s:]+):)?(ttln:(?<ttln>\d+):)?(ttl:(?<ttl>\d+):)?(ttb:(?<ttb>\d+):)?(ttr:(?<ttr>(-)?\d+):)?(ccd:(?<ccd>true|false):)?(sharedKeyEnc:(?<sharedKeyEnc>[^:@]+):)?(pubKeyCS:(?<pubKeyCS>[^:@]+))?(@(?<forAtSign>[^@:\s]*)):(?<atKey>[^:@]((?!:{2})[^@])+)(@(?<atSign>[^@:\s]+))?(:(?<value>.+))?$';
```
Comment (gkc) [inline at_commons/lib/src/verb/syntax.dart]: Note that #166 has a change to notify regex also, take care when merging

Comment (gkc) [reply at_commons/lib/src/verb/syntax.dart]: I've done the merge, please take a look Murali

**Review by gkc — APPROVED** [review]
verb/syntax.dart `static const notify` was modded by #166 as well as #165 ... I've resolved the merge conflict but you might want to double-check

File: at_commons/lib/src/verb/syntax.dart (hunk 1 above)
Comment (murali-shris) [reply at_commons/lib/src/verb/syntax.dart]: thanks Gary

**Pull request merged**